Formula/gnirehtet.rb

```diff
@@ -0,0 +1,67 @@
+class Gnirehtet < Formula
+  desc "Reverse tethering tool for Android"
+  homepage "https://github.com/Genymobile/gnirehtet"
+  url "https://github.com/Genymobile/gnirehtet/archive/v2.3.tar.gz"
+  sha256 "bb2cea28975071b8bdf7bb5dc9536e8119eb0ab113e911cdef7b057bb1ad2582"
+  head "https://github.com/Genymobile/gnirehtet.git"
+
+  depends_on "rust" => :build
+  depends_on "socat" => :test
+
+  resource "java_bundle" do
+    url "https://github.com/Genymobile/gnirehtet/releases/download/v2.3/gnirehtet-java-v2.3.zip"
+    sha256 "93d1d46ee566376596f033832626dd5e89e76c91f2c46d2383735937b7d3b8b0"
+  end
+
+  def install
+    resource("java_bundle").stage { libexec.install "gnirehtet.apk" }
+
+    system "cargo", "install", "--root", libexec, "--path", "relay-rust"
+    mv "#{libexec}/bin/gnirehtet", "#{libexec}/gnirehtet"
+
+    (bin/"gnirehtet").write <<~EOS
```

Comment:
Why would we replace upstream's binary with our own wrapper script?

Reply:
The upstream's binary assumes the `.apk` is in the same directory as the one from which you're  running the command. The install command is basically a wrapper for `adb install -r gnirehtet.apk`.

If you get rid of the wrapper script and run the `gnirehtet` from any directory other than one that contains `gnirehtet.apk`, you'll get this response. 
```
11:51 $ gnirehtet install
2019-11-11 11:51:35.973 INFO Main: Installing gnirehtet client...
Performing Streamed Install
adb: failed to stat gnirehtet.apk: No such file or directory
2019-11-11 11:51:35.995 ERROR Main: Execution error: Command adb ["install", "-r", "gnirehtet.apk"] returned with value 1
```

This is a known issue (https://github.com/Genymobile/gnirehtet/issues/90), but when they close it, I'll be able to remove this wrapper workaround.